model/src/main/java/org/cbioportal/genome_nexus/model/GenomicLocation.java

```diff
@@ -7,6 +7,7 @@
     private Integer end;
     private String referenceAllele;
     private String variantAllele;
+    private String originalInput;
```

Comment:
Not sure if `originalInput` should be part of `GenomicLocation` model. Technically input can be anything, right? Might be better to have a mapping somewhere else instead of tying the input to the model.

Reply:
So the POST accepts a list of `GenomicLocation` objects.. this was my attempt to retain the original input as provided to the POST method since in the service layer the `GenomicLocation` objects are converted into their HGVSg format representations and the `variantAnnotation` services basically loses knowledge of what the original input might have been in the GenomicLocation objects.

Do you have suggestions for how this might be implemented better? @onursumer

Reply:
We can probably add a separate POST which accepts a list of strings instead of genomic location objects. With this change, will we always have to provide the original input value for POST queries from now on?

Reply:
@onursumer So the string representations of the `GenomicLocation` objects? Yeah, I think that would work.. will test first of course but I want to say yes it would.

Reply:
actually I think that some additional changes would be required as well to prevent the normalization of the input genomic location string

Reply:
so we may still run into the case where the start position provided in the input does not match the start position in the annotation json response

Reply:
@onursumer I confirmed that the same issue persists. To make it work this way we would need to skip the step where the genomic location is normalized by the NotationConverter. This is the step that is causing the mismatch between the start positions of the input query and the returned annotation JSON. If there is a common prefix in the reference allele and the variant allele then it is removed and the start position is adjusted accordingly.

Reply:
I guess we can keep it this way if it is too much work to separate the input field from the model